docs/integrations/adding-your-own-connectors.md

```diff
@@ -0,0 +1,49 @@
+# Adding your own connectors
+
+If you'd like to build new connectors and make them part of the pool of pre-built connectors on Airbyte, first a big thank you, and we invite you to check our [contributing guide](https://docs.airbyte.io/contributing/contributing-to-airbyte).
+
+If you'd like to build new connectors, or update existing ones, for your own usage, without contributing to the Airbyte codebase, then that's what open-source is for, and read along. 
+
+## Table of Contents
+
+1. [Airbyte specification](https://docs.airbyte.io/v/dx-to-add-own-connector_suggestion/integrations/adding-your-own-connectors#1-the-airbyte-specification)
+2. [Developing your own connectors](https://docs.airbyte.io/v/dx-to-add-own-connector_suggestion/integrations/adding-your-own-connectors#2-developing-your-own-connectors)
+3. [Adding your connectors on our UI to run them](https://docs.airbyte.io/v/dx-to-add-own-connector_suggestion/integrations/adding-your-own-connectors#3-adding-your-connectors-on-our-ui-to-run-them)
+
+## 1. The Airbyte specification
+
+Before you can start building your own connector, you need to understand [Airbyte's data protocol specification](https://docs.airbyte.io/architecture/airbyte-specification). 
+
+## 2. Developing your own connectors
+
+It's easy to code your own integrations on Airbyte. Here are some links to instruct on how to code new sources and destinations. 
+
+#### **Coding new source connectors:**
+
+* [In Python](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/python-source/README.md)
+* [Based on Singer Taps in Python](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/singer-source/README.md)
+
+#### **Coding new destination connectors:**
```

Comment:
There should also be a step before the UI section that says that in order for Airbyte to be able to leverage a custom integration, it should be published to Dockerhub first (or any image hub that Airbyte can access).

Reply:
I just added a step in section 3 for that, as a first step. Let me know if this is good for you.